tests/Resources/Common.robot

```diff
@@ -156,6 +156,16 @@ Skip If RHODS Version Greater Or Equal Than
        Skip If    condition=${version-check}==True    msg=This test is skipped for RHODS ${version} or greater
     END
 
+Skip If RHODS Is Self-managed
```

Comment:
Does robocop allow a dash in the keyword name?  
Please, consider also capitalize Managed, same as in `Is RHODS Self-Managed`

Reply:
it doesn't complain about the dash. I capitalized it